service/notifier.go

```diff
@@ -161,8 +165,16 @@ func (p *notifier) process(kafkaMsg messaging.Message) error {
 		p.metricScope.Counter(corruptedData)
 		return err
 	}
+	webhook, _ := p.getWebhookForMsg()
+	return p.notifySubscriber(decodedMsg, kafkaMsg, webhook)
+}
 
-	return p.notifySubscriber(decodedMsg, kafkaMsg)
+// TODO: retrieve webhook info for a message
+func (p *notifier) getWebhookForMsg() (*config.Webhook, error) {
+	url, _ := url.Parse("http://localhost:8081/")
```

Comment:
instead of hardcoding, we should use the p.subscriberConfig. 
For your testing, you should update the dev config accordingly: https://github.com/cadence-oss/cadence-notification/blob/master/config/development.yaml#L7

Reply:
do we want to put the webhook config in a database instead of using config file?

Reply:
I think so far we have a simple demo, shall we came up with a more detailed design for the whole project

Reply:
Let’s chat offline for this